src/easyscience/fitting/minimizers/minimizer_dfo.py

```diff
@@ -255,9 +251,7 @@ def _dfo_fit(pars: Dict[str, Parameter], model: Callable, **kwargs):
         """
 
         ## TODO clean when full move to new_variable
```

Comment:
I suppose this TODO can be deleted?

Reply:
This we need to keep.  The if / else statement should be deleted when fully moving to the new variable